### PR TITLE
Finish feature admin approve interviewers, create seeds, fix migratio…

### PR DESCRIPTION
**Pull request by CHILL99**
Finish feature admin approve interviewers, create seeds, fix migration, fix routes with root_path, fix btn login_logout

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -3,9 +3,18 @@ class Admin::UsersController < ApplicationController
...
-    @users = User.all
+    if params[:waiting_approval]
+      @users = User.where.not(approved: true)
+    else
+      @users = User.all
+    end
   end
   
+  def approve
+    User.find(params[:id]).update(approved: true)
```
Comment (Kmalynss) [inline app/controllers/admin/users_controller.rb]: What will happen if user will not be found?

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -3,9 +3,18 @@ class Admin::UsersController < ApplicationController
   before_action :check_admin
 
   def index
-    @users = User.all
+    if params[:waiting_approval]
+      @users = User.where.not(approved: true)
```
Comment (ruba-ruba) [inline app/controllers/admin/users_controller.rb]: it is better to have such logic as a `scope` => `User.waiting_approval`

http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-scope

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (CHILL99) [inline app/controllers/admin/users_controller.rb]: ActiveRecord::RecordNotFound
Couldn't find User

Comment (CHILL99) [inline app/controllers/admin/users_controller.rb]: def approve
    User.find(params[:id]).update(approved: true)
    redirect_to admin_users_path(waiting_approval: true), notice: 'User Approved'
  rescue ActiveRecord::RecordNotFound
    redirect_to admin_users_path, notice: 'Usen not found'
  end

**Pull request merged**